src/Symfony/Component/VarDumper/Tests/Fixtures/VirtualProperty.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Component\VarDumper\Tests\Fixtures;
+
+class VirtualProperty
+{
+    public string $firstName = 'John';
+    public string $lastName = 'Doe';
+
+    public string $fullName {
+        get {
+            return $this->firstName.' '.$this->lastName;
+        }
+    }
+
+    public $noType {
```

Comment:
would be great to have a test with a private property

Reply:
Indeed, I updated this bit and related tests